test/unit/LocalInstallerSpec.ts

```diff
@@ -33,17 +33,19 @@ describe('LocalInstaller install', () => {
 
         it('should pack correct packages', async () => {
             await sut.install();
-            expect(execStub).calledWith(`npm pack ${resolve('b')}`, { cwd: os.tmpdir() });
-            expect(execStub).calledWith(`npm pack ${resolve('c')}`, { cwd: os.tmpdir() });
-            expect(execStub).calledWith(`npm pack ${resolve('/e')}`, { cwd: os.tmpdir() });
+            const maxBuffer = 1024 * 1024 * 10;
+            expect(execStub).calledWith(`npm pack ${resolve('b')}`, { cwd: os.tmpdir(), maxBuffer });
+            expect(execStub).calledWith(`npm pack ${resolve('c')}`, { cwd: os.tmpdir(), maxBuffer });
+            expect(execStub).calledWith(`npm pack ${resolve('/e')}`, { cwd: os.tmpdir(), maxBuffer });
         });
 
         it('should install correct packages', async () => {
             await sut.install();
+            const maxBuffer = 1024 * 1024 * 10;
```

Comment:
Could we move this to the top of the test file? You may make it ALL_CAPS as well (like constant variables in Java)